kubernetes/README.md

```diff
@@ -0,0 +1,71 @@
+# HabitaRPG in Kubernetes
+This is a set of sample kubernetes configuration files to launch
+habitica under aws, both as a single-node web frontent as well as
+multi-node web frontend.
+
+## Prerequisites
+* An AWS account
+* A working kubernetes installation.
+* A basic understanding of how to use kubernetes.
+* A persistant volume for Mongo DB data.
+* Docker images of habitica.
+  + You can use your own, or use mine as specified in the yaml files.
+  + If you use your own, you'll need a fork of the habitrpg github
+    repo and your own dockerhub repo, both of which are free.
+
+
+## Before you begin
+1. Set up Kubernetes.
+2. Create an EBS volume for MongoDB data.
+   + Make a note of the name, you'll need it later.
+
+## Starting MongoDB
+1. Edit mongo.yaml
+  1. Find the volumeID line.
+  2. Change the volume to the one created in the section above.
+2. Run the following commands :
+  - kubectl.sh create -f mongo.yaml
+  - kubectl.sh create -f mongo-service.yaml
+3. Grab some coffee and wait for the mongodb pod to start up. Don't
```

Comment:
Nit: Instead of all of these ☕️coffee☕️ comments, just say "wait" and maybe say what output the user should look for to know it's done.